internal/oci/runtime_oci.go

```diff
@@ -1402,6 +1410,18 @@ func (r *runtimeOCI) runtimeCmd(args ...string) (string, error) {
 
 	err := cmd.Run()
 	if err != nil {
+		stdErrStr := stderr.String()
+		switch {
+		// crun, for most of the commands.
+		case strings.Contains(stdErrStr, "no such process"):
+			fallthrough //nolint:gocritic
```

Comment:
Very interesting to know that go-critic has some opinions about this, which are not necessarily correct.

Related:

- https://github.com/go-critic/go-critic/issues/645

A lot of parser code would use this, and the Go standard library is filled with such language constructs.

I am glad we disabled this nonsense. :+1: